DigiIoT.Maui/DigiIoT.Maui.csproj

```diff
@@ -39,14 +39,14 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="XBeeLibrary.Core" Version="1.0.3" />
+	  <PackageReference Include="XBeeLibrary.Core" Version="1.0.4" />
 	  <PackageReference Include="Plugin.BLE" Version="3.0.0" />
 	  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.3" />
 	</ItemGroup>
 
   <ItemGroup>
-    <None Include="images\nuget_icon.png" Pack="true" PackagePath="\"/>
-    <None Include="..\README.md" Pack="true" PackagePath="\"/>
+    <None Include="images\nuget_icon.png" Pack="true" PackagePath="\" />
+    <None Include="..\README.md" Pack="true" PackagePath="\" />
```

Comment:
Spurious changes.

Reply:
It keeps on changing automatically every time I modify the csproj file. So, I added it to the commit (it is just styling)